bundles/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiBindingConstants.java

```diff
@@ -64,18 +64,45 @@
 
     public static final String CHANNEL_ARTIST = "artist";
     public static final String CHANNEL_TITLE = "title";
+    public static final String CHANNEL_ORIGINALTITLE = "originaltitle";
     public static final String CHANNEL_SHOWTITLE = "showtitle";
     public static final String CHANNEL_ALBUM = "album";
     public static final String CHANNEL_MEDIATYPE = "mediatype";
+    public static final String CHANNEL_MEDIAID = "mediaid";
+    public static final String CHANNEL_MEDIAFILE = "mediafile";
     public static final String CHANNEL_GENRELIST = "genreList";
     public static final String CHANNEL_PVR_CHANNEL = "pvr-channel";
     public static final String CHANNEL_THUMBNAIL = "thumbnail";
     public static final String CHANNEL_FANART = "fanart";
     public static final String CHANNEL_AUDIO_CODEC = "audio-codec";
+    public static final String CHANNEL_AUDIO_CHANNELS = "audio-channels";
+    public static final String CHANNEL_AUDIO_INDEX = "audio-index";
+    public static final String CHANNEL_AUDIO_LANGUAGE = "audio-language";
+    public static final String CHANNEL_AUDIO_NAME = "audio-name";
     public static final String CHANNEL_VIDEO_CODEC = "video-codec";
+    public static final String CHANNEL_VIDEO_INDEX = "video-index";
+    public static final String CHANNEL_VIDEO_HEIGHT = "video-height";
+    public static final String CHANNEL_VIDEO_WIDTH = "video-width";
+    public static final String CHANNEL_SUBTITLE_ENABLED = "subtitle-enabled";
+    public static final String CHANNEL_SUBTITLE_INDEX = "subtitle-index";
+    public static final String CHANNEL_SUBTITLE_LANGUAGE = "subtitle-language";
+    public static final String CHANNEL_SUBTITLE_NAME = "subtitle-name";
+
     public static final String CHANNEL_CURRENTTIME = "currenttime";
     public static final String CHANNEL_CURRENTTIMEPERCENTAGE = "currenttimepercentage";
     public static final String CHANNEL_DURATION = "duration";
+    public static final String CHANNEL_UNIQUEID_IMDB = "uniqueid-imdb";
+    public static final String CHANNEL_UNIQUEID_IMDBTVSHOW = "uniqueid-imdbtvshow";
```

Comment:
`uniqueid-imdbtvshow` is missing from the readme. Also some other new channels are missing from the readme.

Reply:
I have add these into readme and fix line endings of readme.